### DISCUSSION TITLE
Creating new resource hidden by default

### DISCUSSION THREAD

**Discussion by giacomomasseron** [filamentphp/filament#11583] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.37

### How can we help you?

Hi,  
is it possible to create a new resource (with artisan) hidden by default?

Thanks

**Comment by leandrocfe** [comment]
Use Policies
https://filamentphp.com/docs/3.x/panels/resources/getting-started#authorization

**Reply by giacomomasseron** [reply]
What about I create a resource and forget to hide?
I would like to make it hidden by default.

**Reply by leandrocfe** [reply, marked as answer]
*is it possible to create a new resource (with artisan) hidden by default?*
No, I guess

**Comment by zhanang19** [comment]
https://filamentphp.com/docs/3.x/panels/navigation#disabling-resource-or-page-navigation-items
